sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -179,10 +192,11 @@ export type DataFeedRollupSettings =
  */
 export type DataFeedMissingDataPointFillSettings =
   | {
-      fillType: "SmartFilling" | "PreviousValue" | "NoFilling";
+      fillType: "SmartFilling" | "PreviousValue" | "NoFilling" | string;
+      customFillValue?: number;
     }
   | {
-      fillType: "CustomValue";
+      fillType: "CustomValue" | string;
```

Comment:
My expectation is that `customFillValue` is only associated with `fillType: "CustomValue"`. After this change, the "Better typing" is gone. Is there another pattern we should use now to have IntelliSense support?